pkg/tsdb/cloudwatch/routes/log_group_fields_test.go

```diff
@@ -0,0 +1,77 @@
+package routes
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana/pkg/tsdb/cloudwatch/mocks"
+	"github.com/grafana/grafana/pkg/tsdb/cloudwatch/models"
+	"github.com/grafana/grafana/pkg/tsdb/cloudwatch/models/resources"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func Test_log_group_fields_route(t *testing.T) {
```

Comment:
nit: generally speaking things should be camel case
```suggestion
func TestLogGroupFieldsRoute(t *testing.T) {
```